code-challenges.test.js

```diff
@@ -64,8 +105,31 @@ var hand2 = [5, 5, 3, 3, 4]
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
-
-
-
-
+Describe("fullHouse",() =>{
+    test("takes in a an array of 5 numbers and determines a full house",() =>{
+        let hand1 = [5,5,5,3,3,4]
+        let hand2 = [5,5,3,3,4]
+        let hand3 = [5,5,5,5,4]
+        expect(fullHouse(hand1).toEqual([5,5,5,3,3,4]))
+        expect(fullHouse(hand2).toEqual([5,5,3,3,4]))
+        expect(fullHouse(hand3).toEqual([5,5,5,5,4]))
 // b) Create the function that makes the test pass.
+// Create a function that takes in an array of five numbers.
+// I know .map and  .filter makes a new array. .fill and .sort act on an existing array.
+// I know im going to have to have to use a conditional which has to start with an "if" statement.
+// Function named fullHouse that takes in an array.
+// create a new array to store the new list of numbers.
+// for each given number determine if the number is a pair. - having trouble with this 
+// if a pair store into new array.
+// return true or false depending on array of numbers.
+
+const fullHouse = (array)=> {
+    let  newArr = [];
+     for (let i =0; i<array.length; i++){
+     if (array[i] % 2 === 0) {
+     newArr.push(array[i])
+ }
+     }
+ return newArr
+ }
```

Comment:
all I am getting from this is an object {0:4}

You have really good notes with tools listed like .map .filter and forEach none of which are used here. your if statement isn't checking for pairs but if the values are even (like even and odd).